Dockerfile.assisted-service-build

```diff
@@ -6,7 +6,7 @@ ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
 
 USER 0
 
-RUN yum install -y epel-release && yum install -y gcc genisoimage jq git
+RUN yum install -y epel-release && yum install -y gcc genisoimage jq git libvirt-client libvirt-devel
```

Comment:
```suggestion
RUN yum install -y epel-release \
    && yum install -y --setopt=skip_missing_names_on_install=False \
        gcc genisoimage jq git libvirt-client libvirt-devel
```

Reply:
otherwise, it will not verify all those packages exist (and it will just skip everything it couldn't find)